### DISCUSSION TITLE
/Capacitor/Console: File: capacitor-runtime.js - Line 362 - Msg: TypeError: Cannot read property 'ɵprov' of undefined

### DISCUSSION THREAD

**Discussion by ghost** [ionic-team/capacitor#3313] [Q&A]
--E/Capacitor/Console: File: capacitor-runtime.js - Line 362 - Msg: TypeError: Cannot read property 'ɵprov' of undefined.

--E/Capacitor: JavaScript Error: {"type":"js.error","error":{"message":"Uncaught TypeError: Cannot read property 'ɵprov' of 
   undefined","url":"http://localhost/main-es2015.10d628dfe0a7e2cd5491.js","line":1,"col":856445,"errorObject":"{}"}}.

--E/Capacitor/Console: File: http://localhost/main-es2015.10d628dfe0a7e2cd5491.js - Line 1 - Msg: Uncaught TypeError: Cannot 
   read property 'ɵprov' of undefined.

-Getting the above issue when building ionic application in production mode.

**Comment by imhoffd** [comment]
This is just a JavaScript error from your app. I doubt it's related to Capacitor. See if you get the same error with a production build using `ionic serve`.
